[0.367s] Invoking command in '/home/robot33-6/student133_ws/build/test_custom_msg': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake --build /home/robot33-6/student133_ws/build/test_custom_msg -- -j8 -l8
[0.398s] -- Found ament_cmake: 1.3.7 (/opt/ros/humble/share/ament_cmake/cmake)
[0.621s] -- Found geometry_msgs: 4.2.3 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.766s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.766s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.766s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.766s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.766s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.869s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.870s] -- Found ament_lint_auto: 0.12.10 (/opt/ros/humble/share/ament_lint_auto/cmake)
[1.146s] CMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[1.146s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[1.146s]   test_custom_msg --arguments-file
[1.146s]   /home/robot33-6/student133_ws/build/test_custom_msg/rosidl_adapter__arguments__test_custom_msg.json
[1.146s]   --output-dir
[1.146s]   /home/robot33-6/student133_ws/build/test_custom_msg/rosidl_adapter/test_custom_msg
[1.146s]   --output-file
[1.146s]   /home/robot33-6/student133_ws/build/test_custom_msg/rosidl_adapter/test_custom_msg.idls)
[1.146s]   returned error code 1:
[1.146s] 
[1.146s]   Error with: test_custom_msg Triangle geometry_msgs/Point
[1.146s] 
[1.146s]   Traceback (most recent call last):
[1.146s] 
[1.146s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[1.146s]       return _run_code(code, main_globals, None,
[1.147s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[1.147s]       exec(code, run_globals)
[1.147s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[1.147s]       sys.exit(main())
[1.147s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[1.147s]       abs_idl_file = convert_to_idl(
[1.147s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[1.147s]       return convert_msg_to_idl(
[1.147s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[1.147s]       msg = parse_message_string(package_name, input_file.stem, content)
[1.147s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 510, in parse_message_string
[1.147s]       raise InvalidFieldDefinition(line)
[1.147s] 
[1.147s]   rosidl_adapter.parser.InvalidFieldDefinition: geometry_msgs/Point
[1.147s] 
[1.147s] Call Stack (most recent call first):
[1.147s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[1.147s]   CMakeLists.txt:28 (rosidl_generate_interfaces)
[1.147s] 
[1.147s] 
[1.148s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[1.148s] -- Configuring incomplete, errors occurred!
[1.148s] See also "/home/robot33-6/student133_ws/build/test_custom_msg/CMakeFiles/CMakeOutput.log".
[1.204s] Invoked command in '/home/robot33-6/student133_ws/build/test_custom_msg' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake --build /home/robot33-6/student133_ws/build/test_custom_msg -- -j8 -l8
